source/content/guides/secure-development/07-secure-runtime-access.md

```diff
@@ -15,9 +15,9 @@ reviewed: "2022-07-21"
 
 This section provides information on how you can use Secure Runtime Access (SRA) with SSH tunnels to keep your sites secure.
 
-Pantheon’s database services use strong, random passwords and TLS to encrypt communications by default. Customers seeking additional defense can enable Secure Runtime Access.
+Pantheon’s database services use strong, random passwords and TLS to encrypt communications by default. Customers seeking additional defense can enable Secure Runtime Access (SRA).
 
-SRA actively discards attempts to connect to persistent instances like MySQL or SFTP, disregarding the attempt before it reaches the service. When SRA is enabled, the connection attempts to the service will be rejected unless the connection comes through the appropriate [SSH tunnel](/guides/secure-development/ssh-tunnels).
+SRA actively discards attempts to connect to persistent instances like MySQL databases or SFTP, disregarding the attempt before it reaches the service. When SRA is enabled, the connection attempts to the service will be rejected unless the connection comes through the appropriate [SSH tunnel](/guides/secure-development/ssh-tunnels).
```

Comment:
@ryanshoover @Hornswoggles - I added the word `databases` to try to catch the SQL consideration without getting too into it

do you feel like that's enough of a distinction?